addons/sale_margin/models/sale_order.py

```diff
@@ -70,5 +70,5 @@ class SaleOrder(models.Model):
 
     @api.depends('order_line.margin')
     def _product_margin(self):
-        for order in self:
-            order.margin = sum(order.order_line.filtered(lambda r: r.state != 'cancel').mapped('margin'))
+        for order in self.filtered(lambda order: order.state != 'cancel'):
```

Comment:
i know it's difficult to resist, but keep your commits within the scope of the task plz
this has nothing to do with multi-company

Reply:
It came from the order.state/order.company_id removal, but I can drop it if you want :P